proto/rill/runtime/v1/queries.proto

```diff
@@ -308,6 +308,7 @@ message MetricsViewComparisonToplistRequest {
   int64 limit = 10 [(validate.rules).int64.gte = 0];
   int64 offset = 11 [(validate.rules).int64.gte = 0];
   int32 priority = 12;
+  bool approximate = 13;
```

Comment:
Can we flip this to `exact` so the default if not set is to use the approximate implementation?